### ISSUE TITLE
Cannot scrolldown page

### ISSUE THREAD

**Issue by lightla** [WhitePaper233/yukina#1]
This theme cannot scrolldown page

**Comment by WhitePaper233** [comment]
In my testing environment (MacOS+Safari, MacOS+Chrome, Windows 11+Chrome), there was no such issue. Can you provide more detailed information, such as the operating system and browser used and their versions, as well as a detailed description of the problem, such as text descriptions or GIF images, to help me locate the problem

**Comment by teletha** [comment]
I ran into a similar problem, but removing the overflow:hidden to the body element solved the problem. (Windows11, Firefox 133.0.3)

**Comment by WhitePaper233** [comment]
> I ran into a similar problem, but removing the overflow:hidden to the body element solved the problem. (Windows11, Firefox 133.0.3)

I have successfully reproduced the issue in my Windows 11 + Firefox environment. Based on the solution you provided, the issue has been resolved. I'm not entirely sure what role the overflow-hidden class on the body element played at the time, but I now believe it might have been an attribute accidentally left over during debugging, as it doesn't seem to have any other effect.

It’s worth mentioning that I couldn’t reproduce the issue on my MacOS + Firefox Developer Edition, so thank you for the bug report and solution. I will be publishing a new commit to address the issue. My sincere thanks to you!
